runner/runner.go

```diff
@@ -148,6 +148,7 @@ func (r *Runner) executeTest(v models.TestInterface, client *http.Client) (*mode
 
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
+		_ = resp.Body.Close()
```

Comment:
Can we have only one `_ = resp.Body.Close()` call here? If you sure that body needs to be closed despite the error, let's call Close() before the error check.

Reply:
You're right, we can.

And yep, needs to close body despite ReadAll err, it may be err about too large body or something else